decidim-meetings/app/presenters/decidim/meetings/admin_log/questionnaire_presenter.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module AdminLog
+      # This class holds the logic to present a `Decidim::Meetings::Questionnaire`
+      # for the `AdminLog` log.
+      #
+      # Usage should be automatic and you shouldn't need to call this class
+      # directly, but here's an example:
+      #
+      #    action_log = Decidim::ActionLog.last
+      #    view_helpers # => this comes from the views
+      #    InvitePresenter.new(action_log, view_helpers).present
```

Comment:
```suggestion
      #    QuestionnairePresenter.new(action_log, view_helpers).present
```
This comes from the original copied file, let's change this